Tests/L0/SonarQubePostTest/TopLevelOrchestration.ps1

```diff
@@ -0,0 +1,22 @@
+[CmdletBinding()]
+param()
+
+. $PSScriptRoot\..\..\lib\Initialize-Test.ps1
+
+
+# This test ensures the top level functionality of the task is invoked
+
+
+# Arrange   
+Register-Mock InvokeMSBuildRunnerPostTest 
+Register-Mock UploadSummaryMdReport 
+Register-Mock BreakBuildOnQualityGateFailure 
+
+# Act
+. $PSScriptRoot\..\..\..\Tasks\SonarQubePostTest\SonarQubePostTest.ps1    
```

Comment:
By executing the top level task in the test, we make sure all the supporting scripts can be loaded